compiler/one-cmds/utils.py

```diff
@@ -116,7 +116,7 @@ def _add_default_arg(parser):
 
 def is_accumulated_arg(arg, driver):
     if driver == "one-quantize":
-        if arg == "tensor_name" or arg == "scale" or arg == "zero_point":
+        if arg == "tensor_name" or arg == "scale" or arg == "zero_point" or arg == "src_tensor_name" or arg == "dst_tensor_name":
```

Comment:
It could be more readable like this below later.
```python
accumulables = ["tensor_name", "scale", "zero_point", "src_tensor_name", "dst_tensor_name"]
if arg in accumulables:
  ..
```
Or
```python
if arg in ["tensor_name", "scale", "zero_point", "src_tensor_name", "dst_tensor_name"]:
  ..
```

Reply:
@mhs4670go , maybe you can post a PR :)